core/src/main/java/nl/nn/adapterframework/doc/ModelBuilder.java

```diff
@@ -0,0 +1,43 @@
+package nl.nn.adapterframework.doc;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Set;
+
+import lombok.Getter;
+import nl.nn.adapterframework.doc.model.FrankDocModel;
+import nl.nn.adapterframework.doc.objects.SpringBean;
+
+public class ModelBuilder {
+	private @Getter FrankDocModel model;
+
+	public ModelBuilder() {
+		model = new FrankDocModel();
```

Comment:
Wat doet dit? Ik zie 2 static methodes die hier niets mee (kunnen) doen.

Reply:
De code is nog niet af, dat zie je hier. Een FrankDocModel object gaat alle informatie bevatten die je nodig hebt om de Frank!Doc, ibisdoc.xsd, uglify_lookup.xsd en de JSON voor de nieuwe Frank!Doc te genereren. Voordat ik begon heb ik een datamodel getekend. Dat is niet perfect, maar het is goed genoeg als startpunt. Ik heb dit datamodel met je gedeeld onder vermelding van deze pull request.

Reply:
Deze pull request bevat FrankDocModel, FrankDocGroup, FrankElement en FrankAttribute. De andere classes uit het model komen in een latere pull request. In deze pull request maken we alleen de logica waarmee de objecten worden gegenereerd, met unit tests. We gebruiken als testdata classes onder src/test/java met `testtarget` in de package naam. Het maken van het productie FrankDocModel, met alle groepen en elementen en attributes die we nodig hebben, komt later.